src/Components/Todolist.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface TodoListPropsType {
+  todosToShow: Todo[],
+  todoDelete: (todoId: number) => void,
+  deletedId: number,
+  tempTodo: Todo,
+  added: boolean,
+}
+
+export const TodoList: React.FC<TodoListPropsType> = ({
+  todosToShow,
+  todoDelete,
+  deletedId,
+  tempTodo,
+  added,
+}) => {
+  const toShow = todosToShow;
+
+  if (added) {
+    toShow.push(tempTodo);
+  }
```

Comment:
it would be better to use ternary operator
```suggestion
  const todosShown = added ? [...todosToShow, tempTodo] : todosToShow;
```